messages/advise_justification.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
```

Comment:
messaging stuff is fine for now, but eventually we should use `thoth-messaging` by changing its structure to what we have here and in `package-update`

Reply:
is there a card for that?

Reply:
Hmm, I don't think so

Reply:
> messaging stuff is fine for now, but eventually we should use `thoth-messaging` by changing its structure to what we have here and in `package-update`

+1

Reply:
https://trello.com/c/6wBPlbUU let's do it!!